axelor-base/src/main/resources/i18n/messages_fr.csv

```diff
@@ -1,1650 +1,1653 @@
-"key","message","comment","context"
-"${fullName} variants",,,
-"%D : Day","%D : Jour",,
+key,message,comment,context
```

Comment:
double quotes  should not be removed in translation file please take care of this things